src/api/app/views/webui2/webui/patchinfo/show.html.haml

```diff
@@ -0,0 +1,60 @@
+- @pagetitle = @package
+
+- if @patchinfo.block
+  .alert.alert-danger{ role: 'alert' }
+    %i.fa.fa-exclamation-circle
+    %strong Release of this patchinfo is blocked
+    = @patchinfo.block_reason.presence
+.card.mb-3
+  = render partial: 'webui2/webui/package/tabs', locals: { project: @project, package: @package }
+  .card-body
+    .patchinfo.row
+      .col-md-8
+        = patchinfo_header(@patchinfo, @pkg_names)
+        = render partial: 'webui/webui/collapsible_text', locals: { text: @patchinfo.description,
+                                                                    threshold: { text_length: 1000, line_count: 10 },
+                                                                    type: 'description' }
+      .col-md-4
+        = render partial: 'side_elements', locals: { patchinfo: @patchinfo, packager: @packager }
+
+      .col-md-12
+        - if User.current.can_modify?(@package)
+          %ul.list-inline.mb-0
+            %li.list-inline-item
+              = link_to(edit_patchinfo_path(project: @project, package: @package)) do
+                %i.fas.fa-edit.text-secondary
+                Edit patchinfo
+            %li.list-inline-item
+              = link_to('#', class: 'nav-link', data: { toggle: 'modal', target: '#delete-patchinfo-modal' }) do
+                %i.fas.fa-times-circle.text-danger
+                Delete patchinfo
+          = render partial: 'delete_patchinfo_modal', locals: { project: @project, package: @package }
+
+.patchinfo.row
+  - if @patchinfo.message.present?
+    .col-md-12
+      .card.mb-3
+        %h5.card-header Message
+        .card-body
+          = simple_format @patchinfo.message
+
+  .col-md-6
+    .card.mb-3
+      %h5.card-header Fixed bugs
+      .card-body
+        - @patchinfo.issues.each do |issue|
+          .media
+            %i.pt-1.mr-1.fas.fa-bug.text-danger
+            .media-body
+              - link_text = issue[0] == 'CVE' ? issue[1] : "#{issue[0]}##{issue[1]}"
```

Comment:
could you please move it to a helper? I'm quite sure the security team prefer to read here ```CVE-2012-1234``` then ```cve#2012-1234```

Reply:
done!

Reply:
![Screenshot_2019-04-17 Open Build Service(6)](https://user-images.githubusercontent.com/1212806/56302538-42e7dd80-613a-11e9-9d9b-05f5418f6f7b.png)